api/src/main/java/gov/cms/ab2d/api/repository/JobRepository.java

```diff
@@ -7,4 +7,5 @@
 @Repository
```

Comment:
I just realized that this repository is in the `api` module. This needs to be moved to the `domain` module, since both the API and Worker will use this repository.

Reply:
Discussed over slack and this can be completed in a separate jira to refactor code to go into the shared module.